Extensions/PrimitiveDrawing/shapepainterruntimeobject-pixi-renderer.ts

```diff
@@ -20,6 +20,9 @@ namespace gdjs {
      */
     _transformationIsUpToDate = false;
 
+    /** The antialiasing filter */
```

Comment:
This is not a good comment: it adds no context that the variable name does not already carry. It is useless and only slows down the reading of the code - it's counterproductive. Either remove it or rewrite it to something useful, e.g. "An instance of the PIXI FXAA shader that is applied to the render object when antialiasing on the ShapePainterRuntimeObject has been turned on" - although I personally think that this much context can still be implicitly understood from the variable name.